notebooks/official/feature_store/sdk-feature-store-pandas.ipynb

```diff
@@ -30,15 +30,23 @@
       },
```

Comment:
<h4>Feature values from last ingestion</h4><p> </p><p>Reading from the entity type gets feature values from the most recent ingestion.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>